_includes/navbar.html

```diff
@@ -56,6 +56,7 @@
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">About Apache<strong class="caret"></strong></a>
                     <ul class="dropdown-menu">
                         <li><a href="http://www.apache.org">The Apache Software Foundation</a></li>
+                        <li><a href="http://www.apache.org/licenses/">License</a></li>
```

Comment:
Makes sense. Although this section is more "about the Apache Software Foundation", and not abut jclouds itself. I just added it in accordance to: http://www.apache.org/foundation/marks/pmcs.html#navigation